iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -754,5 +762,15 @@ - (void)resetElevationMyPositionChanged {
   self.bm.SetElevationMyPositionChangedCallback(nullptr);
 }
 
+static const uint8_t kMaxChannelValue = 255;
+
+- (dp::Color)getColorFromUIColor:(UIColor *)color {
+    const CGFloat * components = CGColorGetComponents(color.CGColor);
```

Comment:
Why is it better than `[color getRed:&red green:&green blue:&blue alpha:&alpha];` ? It is hard to say how many values were returned and which value is where.